src/main/java/frc/robot/SuperStructure.java

```diff
@@ -109,13 +113,15 @@ public boolean atTargetPositions() {
     public Command followPositions(DoubleSupplier elevatorPosition, DoubleSupplier shoulderPosition, DoubleSupplier wristPosition) {
         return Commands.parallel(
                 arm.followPositions(shoulderPosition, wristPosition),
-                elevator.followPosition(elevatorPosition));
+                elevator.followPosition(elevatorPosition),
+                run(() -> {}));
     }
 
     public Command goToPositions(double elevatorPosition, double shoulderPosition, double wristPosition) {
         return Commands.parallel(
                 elevator.goToPosition(elevatorPosition),
-                arm.goToPositions(shoulderPosition, wristPosition));
+                arm.goToPositions(shoulderPosition, wristPosition),
+                run(() -> {}));
```

Comment:
I was wrong here. This should actually be `runOnce`. Otherwise the parallel command won't end.